lib/restrictedsession/bytecode/README.md

```diff
@@ -1,3 +0,0 @@
-## BPF Bytecode
```

Comment:
This too - did you intend to delete this?

Reply:
No, this was not intended. Thanks.